src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -194,8 +210,40 @@ class DocumentStatisticsTrainingOnlyPolicy {
 template <
     size_t MaxNumTrainingItemsV=std::numeric_limits<size_t>::max()
 >
-using DocumentStatisticsEstimator           = Components::TrainingOnlyEstimatorImpl<Details::DocumentStatisticsTrainingOnlyPolicy, MaxNumTrainingItemsV>;
+class DocumentStatisticsEstimator : public TrainingOnlyEstimatorImpl<Details::DocumentStatisticsTrainingOnlyPolicy, MaxNumTrainingItemsV> {
+public:
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Types
+    // |
+    // ----------------------------------------------------------------------
+    using BaseType                          = TrainingOnlyEstimatorImpl<Details::DocumentStatisticsTrainingOnlyPolicy, MaxNumTrainingItemsV>;
+    using StringDecorator                   = Details::DocumentStatisticsTrainingOnlyPolicy::StringDecorator;
+    using IndexMap                          = Details::DocumentStatisticsTrainingOnlyPolicy::IndexMap;
+    using AnalyzerMethod                    = Details::DocumentStatisticsTrainingOnlyPolicy::AnalyzerMethod;
+    using RegexT                            = Details::DocumentStatisticsTrainingOnlyPolicy::RegexT;
```

Comment:
>using RegexT                            = Details::DocumentStatisticsTrainingOnlyPolicy::RegexT; [](start = 4, length = 96)

It doesn't look like this is used publicly

Reply:
have put it privately